modules/ROOT/pages/cassandra/cassandra-studio.adoc

```diff
@@ -0,0 +1,112 @@
+= Studio - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Install Cassandra
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click Login in Anypoint Exchange and supply your Anypoint Platform username and password.
+. Search for the Cassandra connector and click Install.
+. Follow the prompts to install the connector.
+
+== Configure Cassandra
+. Drag and drop the operation connector to the Studio canvas.
++
+[%header%autowidth.spread]
+|===
+|Operation |Description
+|Add New Column a|Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
```

Comment:
How does customer know what a keyspace name is or where to obtain the value? A hint here would be helpful.